applications/TrilinosApplication/trilinos_space.h

```diff
@@ -31,6 +31,9 @@
 #include <EpetraExt_VectorIn.h>
 #include <EpetraExt_RowMatrixOut.h>
 #include <EpetraExt_MultiVectorOut.h>
+#include <EpetraExt_MatrixMatrix.h>
+
+// NOTE: Epetra will be replaced with Tpetra in the future, there is an intermediate interface called Xpetra, which uses the interface of Toetra but allows to use Epetra
```

Comment:
Not sure if it is worth the effort, is TPetra better?

I would rather work towards our own CSR matrix to drop Trilinos

Reply:
(Just a comment, but according to the official benchmarks is better...)

Reply:
Comment removed